files/en-us/web/api/domrectreadonly/fromrect/index.html

```diff
@@ -18,20 +18,20 @@
 
 <h2 id="Syntax">Syntax</h2>
 
-<pre class="syntaxbox notranslate">var domRect = DOMRectReadOnly.fromRect(rectangle)</pre>
+<pre class="syntaxbox notranslate">let domRect = DOMRectReadOnly.fromRect(<var>rectangle</var>)</pre>
```

Comment:
By and large the syntax section of pages isn't a precise reflection of js, more how something is to be expected to be used with correctly names params and returns. Also by adding `let` it implies `const` isn't applicable (whereas it is). However edits here highlight the vars that are missing:

```suggestion
<pre class="syntaxbox notranslate">var <var>domRect</var> = DOMRectReadOnly.fromRect(<var>rectangle</var>)</pre>
```

Reply:
I just remember seeing “`let` is the new `var`” everywhere when the ES6 syntax started to become available.  I remember reading that let follows more sensible rules for scope, and so it should be preferred over `var`.  That’s all I was going for.  I wasn’t intending to imply anything about `const`.

Reply:
(I can’t believe I missed wrapping `domRect` in an HTML `var` element, though! Thanks.)

So, I forgot to ask for clarity: **Is it really preferred on MDN to use `var`?**  

I was under the (strong) impression that it should be avoided (except for things like legacy code, blah blah, where the different scoping rules might introduce unintended consequences...).

Reply:
So syntax boxes are in a weird category of being not quite the js you would write, but an example of usage. We probably do need to clarify precise guidelines (and in doing so conventions we use now could change).

In reference to `var` vs `let` vs `const` when it comes to examples (as in the page section examples) which ever seems appropriate is absolutely fine...

And there's every chance we'll change the syntax box conventions in the future, but for now it's probably best to stick with `var` anticipating a wider discussion :)

Reply:
Thanks, @Rumyra!